gravitee-node-monitoring/src/main/java/io/gravitee/node/monitoring/healthcheck/NodeHealthCheckManagementEndpoint.java

```diff
@@ -18,36 +18,36 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import io.gravitee.common.http.HttpHeaders;
 import io.gravitee.common.http.HttpMethod;
 import io.gravitee.common.http.HttpStatusCode;
 import io.gravitee.common.http.MediaType;
 import io.gravitee.node.api.healthcheck.Probe;
+import io.gravitee.node.api.healthcheck.ProbeRegistry;
 import io.gravitee.node.api.healthcheck.Result;
 import io.gravitee.node.management.http.endpoint.ManagementEndpoint;
+import io.netty.handler.codec.http.HttpHeaderNames;
 import io.vertx.core.http.HttpServerResponse;
 import io.vertx.core.json.jackson.DatabindCodec;
 import io.vertx.ext.web.RoutingContext;
 import java.util.Map;
 import java.util.stream.Collectors;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import lombok.extern.slf4j.Slf4j;
 
 /**
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
+@Slf4j
 public class NodeHealthCheckManagementEndpoint implements ManagementEndpoint {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(NodeHealthCheckManagementEndpoint.class);
-
-    private NodeHealthCheckThread registry;
+    private final ProbeRegistry probeRegistry;
 
     final ObjectMapper objectMapper;
 
     public static final String PROBE_FILTER = "probes";
 
-    public NodeHealthCheckManagementEndpoint() {
+    public NodeHealthCheckManagementEndpoint(ProbeRegistry probeRegistry) {
+        this.probeRegistry = probeRegistry;
         objectMapper = DatabindCodec.prettyMapper();
```

Comment:
I am still wondering if we should prefer a simple objectmapper instead of pretty one.

Reply:
I finally reuse the already existing mapper like it is made in `NodeMonitorManagementEndpoint`. 👍